src/app/pages/statistics/consumption/statistics-consumption.component.ts

```diff
@@ -57,10 +59,8 @@ export class StatisticsConsumptionComponent implements OnInit {
   public ngOnInit(): void {
     let filterDef: TableFilterDef;
 
-    filterDef = new StartDateFilter().getFilterDef();
-    this.allFiltersDef.push(filterDef);
-
-    filterDef = new EndDateFilter().getFilterDef();
+    filterDef = new DateRangeFilter().getFilterDef();
+    filterDef.timePicker24Hour = this.language !== 'en';
```

Comment:
Kind of a hack, locale should be passed to the date/time picker and it should handle the way the hours are formatted or find another way the set the 24 hours (check moment API ?)